src/components/Modal/Modal.stories.jsx

```diff
@@ -20,6 +20,41 @@ const Template = args => {
   );
 };
 
+const CrisisTemplate = args => {
+  const [visible, setVisible] = useState(args.visible);
+  const onClose = () => setVisible(!visible);
+  const openModal = () => setVisible(true);
+  return (
+    <>
+      <div
+        className="va-crisis-line"
+        style={{ position: 'static', height: '600px', maxWidth: 'none' }}
```

Comment:
`maxWidth` is set here to remove what the `va-crisis-line` CSS is adding. We can't actually remove it from the CSS because that is used in teamsite pages where the React component can't be used.